main.sh

```diff
@@ -32,8 +32,8 @@ if [ "${BASH_SOURCE[0]}" != "${0}" ]; then
     __b3bp_tmp_source_idx=1
   fi
 else
-  [ ! -z "${__usage+x}" ] && unset __usage
-  [ ! -z "${__helptext+x}" ] && unset __helptext
+  [ ! -z "${__usage+x}" ] && unset -v __usage
+  [ ! -z "${__helptext+x}" ] && unset -v __helptext
```

Comment:
One nit: Is there a reason to be using `! -z` rather than `-n`? It seems like a double negative to me, wouldn't we rather just use `-n`? Or is there a subtlety I'm missing here?